legend-engine-configuration/src/main/java/org/finos/legend/engine/plan/generation/extension/LegendPlanGeneratorExtension.java

```diff
@@ -44,10 +44,12 @@ public RichIterable<? extends Root_meta_pure_extension_Extension> getExtraExtens
     {
         MutableList<Root_meta_pure_extension_Extension> pureExtensions = Lists.mutable.empty();
 
-        RichIterable<Root_meta_external_shared_format_ExternalFormatExtension> planGenerationExtensions = LazyIterate.collect(ExternalFormatPlanGenerationExtensionLoader.extensions().values(), ext -> ext.getPureExtension(pureModel.getExecutionSupport()));
-        pureExtensions.addAll(core_pure_binding_extension.Root_meta_external_shared_format_routerExtensions_String_1__ExternalFormatExtension_MANY__Extension_MANY_("externalFormat", planGenerationExtensions, pureModel.getExecutionSupport()).toList());
+        pureExtensions.add(Root_meta_external_shared_format_externalFormatExtension__Extension_1_(pureModel.getExecutionSupport()));
```

Comment:
Should we load extensions dynamically based on class path or we want compile level checks?

Reply:
I think we should load dynamically so that any developer building a binding can use it without make edits to this file